homeassistant/components/html5/strings.json

```
@@ -27,6 +27,28 @@
     }
   },
   "services": {
+    "notify": {
+      "name": "Send a notification with html5",
+      "description": "Sends a notification message using the html5 service.",
+      "fields": {
+        "message": {
+          "name": "Message",
+          "description": "The message to be sent."
+        },
+        "title": {
+          "name": "Title",
+          "description": "The title of the message (optional)."
```

In the UI an optional field has a check box next to it so we don't need to add that to the description.
Also the data field is optional as well – IIRC.

```suggestion
          "description": "The title of the message."
```